### DISCUSSION TITLE
Devtools on Vue 2

### DISCUSSION THREAD

**Discussion by Djaler** [vuejs/pinia#857] [Help and Questions] [answered]
With the same code
![image](https://user-images.githubusercontent.com/7964583/143998537-9a8fd6cc-b02d-48be-99c6-045c26602bda.png)
mutation info in devtools in Vue 3 looks like
![image](https://user-images.githubusercontent.com/7964583/143998462-8eea6fb7-7929-4bfe-b5b9-791e71a8f442.png)
but in Vue 2 it's just
![image](https://user-images.githubusercontent.com/7964583/143998491-d56980ee-5522-4d2a-855f-cca508bc3d81.png)
 which is not so useful for debugging

**Comment by posva** [comment, marked as answer]
Yeah, this is currently lacking: https://github.com/posva/pinia/issues/859
